app/controllers/application_controller.rb

```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```

Comment:
What purpose does this serve?

This will get spammy.  Any request with an auth token header to a controller action that is not exempt from authentication will print this.